python/README.md

```diff
@@ -0,0 +1,21 @@
+# Configuration Script
+This script helps you configure your AWS credentials into the requiered files and initialize terraform backends.
+
+To run this script you need to configure first a virtual environment:
+
+```
+cd terraform-academy/python
+pip3 install virtualenv
```

Comment:
https://twitter.com/alicegoldfuss/status/1116150186827337729 :trollface:

Reply:
Spent quite some time fighting with the virtualenvs and pylint... not proud of it hahaha